### PR TITLE
Removed eslint-config-node-services and mocha-eslint deps

### PR DESCRIPTION
**Pull request by hknustwmf**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -156,16 +156,16 @@ const nameSpaceIds = {
...
 nameSpaceIds.Image = nameSpaceIds.File;
 nameSpaceIds.ImageTalk = nameSpaceIds.FileTalk;
 
 /* Define a boolean is<ns> function for every namespace */
 for (const ns in nameSpaceIds) {
     if (Object.prototype.hasOwnProperty.call(nameSpaceIds, ns)) {
-        (function(id) {
-            Namespace.prototype[`is${ns}`] = function() {
+        (function (id) {
+            Namespace.prototype[`is${ns}`] = function () {
```
Comment (Pchelolo) [inline lib/index.js]: since we are touching these can as well convert to arrow syntax

File: lib/mediawiki.Title.phpCharToUpper.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
-"use strict";
+'use strict';
 
 // ESLint suggests unquoting some object keys, which would render the file unparseable by Opera 12.
 /* eslint-disable quote-props */
-(function() {
+(function () {
```
Comment (Pchelolo) [inline lib/mediawiki.Title.phpCharToUpper.js]: idem

File: lib/utils.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ module.exports = {};
  * @param {string} byteClass
  * @return {string}
  */
-module.exports.convertByteClassToUnicodeClass = function(byteClass) {
+module.exports.convertByteClassToUnicodeClass = function (byteClass) {
```
Comment (Pchelolo) [inline lib/utils.js]: idem

**Comment by Pchelolo** [comment]
couple of nits, otherwise lgtm

**Comment by Pchelolo** [comment]
Please update travis to use 6, 10 and 'node'

File: lib/index.js (hunk 1 above)
Comment (hknustwmf) [reply lib/index.js]: Arrow syntax is discouraged with prototype

File: lib/mediawiki.Title.phpCharToUpper.js (hunk 2 above)
Comment (hknustwmf) [reply lib/mediawiki.Title.phpCharToUpper.js]: I tried. Changing the IFFE to arrow causes a lint parsing error.

File: lib/utils.js (hunk 3 above)
Comment (hknustwmf) [reply lib/utils.js]: Done

**Pull request merged**